docs/jwst/references_general/references_general.rst

```diff
@@ -572,7 +572,7 @@ Bit  Value         Name              Description
 1    2             SATURATED         Pixel saturated during exposure
 2    4             JUMP_DET          Jump detected during exposure
 3    8             DROPOUT           Data lost in transmission
-4    16            RESERVED
+4    16            OUTLIER           Removed by outlier detection
```

Comment:
Minor nit, but technically the pixel isn't being removed by `outlier_detection`, it's only being flagged. It's then up to other steps to decide whether or not to remove it. So perhaps "Flagged by outlier detection".